### PR TITLE
[master] Auto-update dependencies

### PR DESCRIPTION
**Pull request by mattmoor**
Produced via:
  `./hack/update-deps.sh --upgrade && ./hack/update-codegen.sh`
/assign maximilien rhuss
/cc maximilien rhuss

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -20,10 +20,10 @@ require (
 	k8s.io/apimachinery v0.17.0
 	k8s.io/cli-runtime v0.17.0
 	k8s.io/client-go v0.17.0
-	knative.dev/eventing v0.13.6
-	knative.dev/pkg v0.0.0-20200409225505-9147309d5366
-	knative.dev/serving v0.13.2
-	knative.dev/test-infra v0.0.0-20200409221904-5e04d955cdb9
+	knative.dev/eventing v0.13.7
```
Comment (matzew) [inline go.mod:23]: there seems to be a merge conflict ? 

close this? since it is generally generated by a bot /cc @rhuss  ?

**Comment by rhuss** [comment]
@matzew I'd like to keep that PR open until we have release 0.14.0 as we now have pinned already the dep to 0.14 serving/eventing and so want to "trap" all unwanted auto-updates with this issue (otherwise just a new PR is openend on a serving/eventing change upstream)

After 0.14.0 is relased we should close this PR

**Comment by rhuss** [comment]
I will work on this tomorrow.

**Pull request closed without merging**